config.yaml

```diff
@@ -0,0 +1,8 @@
+DD_APP_KEY: <Datadog Application Key>
+DD_API_KEY: <Datadog API key>
+DD_SITE: datadoghq.com
+DD_TAG: <TODO: Implement tag filtering>
+N9_DS: 
+N9_DS_PROJECT: 
```

Comment:
This makes some of the already long lines in the script even longer. While I like the challenge of splitting up lines so they look nice, I think I'd rather leave it like this since there are so few configuration options and it shouldn't really confuse people.